static/js/latest-news.js

```diff
@@ -0,0 +1,112 @@
+function formatDate(date) {
+  const parsedDate = new Date(date);
+
+  const day = parsedDate.getDate();
+  const monthIndex = parsedDate.getMonth();
+  const year = parsedDate.getFullYear();
+
+  return year + "年" + (monthIndex + 1) + "月" + day + "日";
+}
+
+function htmlForLatestArticles(articles) {
+  const articlesTree = document.createDocumentFragment();
+  for (let i = 0; i < articles.length; i++) {
+    const article = articles[i];
+    const link = `<a href="/blog/${article.slug}">${
+      article.title.rendered
+    }</a>`;
+    const header = `<h4>${link}</h4>`;
+    const date = `<p class="u-no-padding--top">
+        <em>
+          <time pubdate datetime="${article.date}">
+            ${formatDate(article.date)}
+          </time>
+        </em>
+      </p>`;
+    const div = document.createElement("div");
+    div.classList.add("col-3");
+    div.innerHTML = ` 
+        ${header}
+        ${date}
+      `;
+
+    articlesTree.appendChild(div);
+  }
+  return articlesTree;
+}
+
+function htmlForLatestPinnedArticle(article) {
+  const articlesTree = document.createDocumentFragment();
+
+  // "Featured"
+  const header = "<h3>注目のニュース</h3>";
+
+  const link = `<a href="/blog/${article.slug}">${article.title.rendered}</a>`;
+  const linkHeader = `<h4>${link}</h4>`;
+
+  const date = `<p class="u-no-padding--top">
+      <em>
+        <time pubdate datetime="${article.date}">
+            ${formatDate(article.date)}
+        </time>
+      </em>
+    </p>`;
+
+  const innerDiv = `
+      ${linkHeader}
+      ${date}
+    `;
+
+  const div = document.createElement("div");
+  div.innerHTML = `
+    ${header}
+    ${innerDiv}
+  `;
+
+  articlesTree.appendChild(div);
+
+  return articlesTree;
+}
+
+const createReqListenerForDomContainer = (
+  latestNewsContainer,
+  spotlightContainer
+) => e => {
```

Comment:
`e` is not very descriptive and it's hard to know what it means until you read further in to the function. Please change to `event` (as I assume that's what it means).